extension.json

```diff
@@ -194,18 +315,6 @@
       {
         "type": "remove",
         "propertyPath": "instances[].edgeConfigInputMethod"
-      },
-      {
-        "type": "remove",
-        "propertyPath": "instances[].sandbox"
-      },
-      {
-        "type": "remove",
-        "propertyPath": "instances[].stagingSandbox"
-      },
-      {
-        "type": "remove",
-        "propertyPath": "instances[].developmentSandbox"
```

Comment:
Why get rid of these?

Reply:
I deleted them while trying something out and forgot to add them back.